app/services/pii/store.rb

```diff
@@ -0,0 +1,40 @@
+module Pii
+  class Store
```

Comment:
So `Pii::Store` is probably not the best name for this class, I couldn't think of anything better (it's just a nice wrapper around the redis session store), would love some suggestions